beta-20160602.md

```diff
@@ -0,0 +1,29 @@
+---
+title: What's New in beta-20160602
+toc: false
+---
+
+## Jun 2, 2016
+
+### New Features
+
+* String literals can now be parsed as `DATE`, `TIMESTAMP`, `TIMESTAMPTZ` or `INTERVAL` without an explicit cast. [#6925](https://github.com/cockroachdb/cockroach/pull/6925)
+* Floor division is now supported with a new operator `//`. [#6642](https://github.com/cockroachdb/cockroach/pull/6642)
+* Sub-queries are now allowed in `LIMIT`, `OFFSET`, and `RETURNING` expressions. [#6735](https://github.com/cockroachdb/cockroach/pull/6735)
+
+### Bug Fixes
+
+* Fixed a missing error check that could result in inconsistences when transactions conflict. [#6899](https://github.com/cockroachdb/cockroach/pull/6899)
+* Indexes and tables that are referenced by foreign keys may not be dropped. [#6832](https://github.com/cockroachdb/cockroach/pull/6832)
```

Comment:
Foreign keys aren't yet supported, right? This is just preparatory work? If so, perhaps we should add a disclaimer so users don't get the wrong impression here. 

If you agree, I'm happy to add this when I add docs links/updates.